### ISSUE TITLE
Item start time occasionally incorrect in the Show Manager

### ISSUE THREAD

**Issue by mroskamp** [mcallegari/qlcplus#723]
In the Show Manager of QLC+ (4.10.2a), when you have a sequence and open the "Item start time and duration" dialog, the leading zeros are trimmed, making the time incorrect if the millisecond is less than .1 seconds. For example (see below), if the start time is 49.08 second, it actually displays 49.8 seconds.

![image](https://cloud.githubusercontent.com/assets/4029047/11831325/a7b347b6-a37b-11e5-9acd-954fcb82a0fa.png)

**Comment by mcallegari** [comment]
10.10.2a doesn't exist. You meant 4.10.2a
4.10.2a is not the latest build. Please use 4.10.2b

I am aware there are still cases of failure in the time display.
Can you please indicate the exact steps from an empty project to quickly reproduce the issue.
Or share an existing workspace...which can't be attached on GitHub. That's why I always suggest to use the forum first and then in case we move here.

**Comment by mroskamp** [comment]
Yes, the version was a typo. 4.10.2a is the version I'm runnning. I have a production this weekend, after which I'll update and try to reproduce the issue and post in the forum for further discussion.

**Comment by plugz** [comment]
@mcallegari While I'm at it, are you OK with changing the precision down to 1ms instead of 10ms ?

**Comment by mcallegari** [comment]
From the UI perspective it would make sense, but keep in mind that the engine (granularity is 20ms)[https://github.com/mcallegari/qlcplus/blob/master/engine/src/mastertimer.cpp#L45], so all the time values less than 20ms will be rounded (or ignored) at some point in the Function::write methods.

It makes sense instead in both areas when using a BPM based system (which I intend to implement later this year).

In conclusion, let's get rid of this "centisecond" measurement, but be prepared to reply to questions in the forum like "342ms are actually 340ms"

**Comment by plugz** [comment]
The engine tick happens every 20ms yes.

This is not an issue, and having a precision down to the ms will improve things for things running for a long time.



> In conclusion, let's get rid of this "centisecond" measurement, but be prepared to reply to questions in the forum like "342ms are actually 340ms" (or even worse, 333ms turns into 320ms, or 340ms probably)

The issue would already be present: I can select duration of 30ms for a matrix.
This commit fixed the issue with a 30ms duration: f5b5b86fcb709f0073a8c16744bab1c1dcb8a463

The 4 first step changes should happen at:
30, 60, 90, 120
The step changes, without the commit, happened at:
40, 80, 120, 160
After the commit, they actually happen at:
40, 60, 100, 120

So 4 steps per 120ms.
The actual step change time is sometimes shifted, but in the long run, the time does not drift. That's what's important.

I don't know if I'm clear enough.

**Comment by mcallegari** [comment]
That's clear to me :wink:
